pkg/client/cmd/run.go

```diff
@@ -51,6 +52,7 @@ func NewCmdRun(rootCmdOptions *RootCmdOptions) *cobra.Command {
 	}
 
 	cmd.Flags().StringVarP(&options.Language, "language", "l", "", "Programming Language used to write the file")
+	cmd.Flags().StringVarP(&options.IntegrationName, "integrationName", "i", "", "The integration name")
```

Comment:
Maybe we can use just `name` here to set it as `--name`, without short flag (`i` is not easy to remember, `n` is used for namespace).